app/controllers/api/v1/profiles_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Api::V1
+  class ProfilesController < ApiController
+
+    def index
+      if params[:uids].present?
+        profiles = Profile.where(uid: params[:uids])
+      else
+        profiles = Profile.all
```

Comment:
I think it's more about permissions, do we have separate stories for that? 

I'm sorry for maybe asking redundant questions, I'm still not caught up properly.